tests/afterSend.test.js

```diff
@@ -18,18 +24,12 @@ describe('Using the after() method', function() {
 
   before(cleanupLog);
 
-  it('should intercept the generated .html and respond with the same .html', function(done) {
-    request(app)
-      .get('/README.md')
-      .expect(200)
-      .end(done);
+  it('should intercept the generated .html and respond with the same .html', function() {
+    return expect('/README.md', 'to yield response', 200);
   });
 
-  it('verify log file exist', function(done) {
-    fs.exists('body.log', function(exist) {
-      expect(exist).to.equal(true);
-      done();
-    });
+  it('verify log file exist', function() {
+    return expect(fs.existsSync('body.log'), 'to be true');
```

Comment:
Seems like a good idea. I prefer doing this in another PR, as this is already too big :-) If you are okay with that, I'll make a new issue for that, and fix it later.